openlibrary/plugins/upstream/code.py

```diff
@@ -33,6 +35,23 @@ def GET(self):
         raise web.seeother(host + '/static' + web.ctx.path)
 
 
+class view(core.view):
+    def GET(self, path):
+        if web.re_compile(r'/people/[^/]+/lists/OL[\d]+L(/[^/]+){0,1}').match(path):
```

Comment:
Yeah, I'm seeing that history pages aren't being correctly handled in this PR...

I was trying to avoid making a major change to a common template.  Will remove this in favor of template change.